packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -122,6 +136,33 @@ const mapStyle: React.CSSProperties = {
   pointerEvents: 'auto',
 };
 
+/**
+ * The following code and styles are for demonstration purposes
+ * at this point. After #1671 is merged, we can implement these
+ * menu buttons and their associated panels for real.
+ */
+const buttonStyles: React.CSSProperties = {
+  alignItems: 'center',
+  background: 'transparent',
+  borderColor: 'transparent',
+  display: 'flex',
+  fontSize: 16,
```

Comment:
I know this predates this PR, but this should really be a relative font size. It might be best to make a follow-up issue for this.

Reply:
Here's the follow up issue for this: https://github.com/VEuPathDB/web-monorepo/issues/274